requirements/system/user-client.tex

```diff
@@ -1,9 +1,172 @@
+    \item [UCFR12] Checking the state of the entire server
+  \end{itemize}
+
+  %\subsubsection{Non Functional Requirements} 
+  %\begin{itemize}
+    %\item [NFRUC01] Responses from the server must take less 10ms
+    %\item [NFRUC02] Users can't queue more than 5 jobs at the same time 
+  %\end{itemize}
+
+  \subsubsection{Optional Requirements} 
+  \begin{itemize}
+    \item [UCOR01] Saving a private image for future usage
+    \item [UCOR02] Scheduling a job to run at a specific time
+    %\item [UCOR03] Choosing a preferred container to use
+    \item [UCOR03] Checking the estimated waiting time
+    \item [UCOR04] Checking the estimated time of arrival
+  \end{itemize}
+
+  \subsubsection{Product Data} 
+  \begin{itemize}
+    \item [UCPD01] Log in information
+    \item [UCPD02] Configuration files
+    \item [UCPD03] Currently running jobs
+    \item [UCPD04] Output of finished jobs
+  \end{itemize}
+
+  \subsubsection{Requirement Details} 
+\begin{itemize}
+	\item UCFR01 Logging in to the central server
+	\begin{itemize}
+		\item Goal: Ability to register and log into a User Account.
+		\item Importance: Primary
+		\item Precondition: User needs to have an account.
+		\item Post condition (success): The User is logged into a User Account.
+		\item Post condition (fail):
+		\begin{itemize}  
+			\item The User is not logged into a User Account.
+			\item An appropriate error message is displayed.
+		\end{itemize}
+		\item Triggering Event: The User starts the Application.
+		\item Description:
+		\begin{itemize}
+			\item   The User can log into a User Account by entering the correct username and a matching password in the CLI
+		\end{itemize}
+	\end{itemize}
+	
+	\item UCFR02 Logging out from the central server
+	\begin{itemize}
+		\item Goal: Ability to log out from of user account.
+		\item Importance: Primary
+		\item Precondition: User is logged into a user account.
+		\item Post condition (success): The User is no longer logged into a user account.
+		\item Post condition (fail):
+		\begin{itemize}
+			\item The user is not logged out.
+			\item An appropriate error message is displayed.
+		\end{itemize}
+		\item Triggering Event: The user shuts down the application.
+		\item Description:
+		\begin{itemize}
+			\item The User will be logged out by shutting down the application.
+		\end{itemize}
+	\end{itemize}
+	
+	\item UCFR03 Queueing new jobs
+	\begin{itemize}
+		\item Goal: Allow User to queue jobs.
+		\item Importance: Primary
+		\item Precondition: User is logged into a user account.
+		\item Post condition (success): The job is queued.
+		\item Post condition (fail):
+		\begin{itemize}
+			\item The job is not queued.
+			\item An appropriate error message is displayed.
+		\end{itemize}
+		\item Triggering Event: The User requests a "job add" command
+		\item Description:
+		\begin{itemize}
+			\item  Queueing a job can be toggled by the ”add command” in the CLI.
```

Comment:
We should stick to one symbol for Quotation marks.

Reply:
I'm not sure what you mean by "one symbol for Quotation marks"

Reply:
” vs "one of those, you used them both in different sentences.